### DISCUSSION TITLE
🎉 Congrats guys for 4.0 version!

### DISCUSSION THREAD

**Discussion by jmglezgz** [abpframework/abp#6483] [General]
Hi guys,
Sorry for disturb in the issues channel with a celebration! the intention was always laudable.

 I'm trying here now. You can delete it again if it's not the correct channel, however, I would appreciate it if you can say me where the place where open this kind of discussion.

Thanks so much,
Jose

**Comment by hikalkan** [comment]
Thanks @jmglezgz for your message :)
Such messages motivate us. No problem to write here.

**Comment by jmglezgz** [comment]
Thanks guys, I'm learning a lot from your knowledge! and working hard on getting myself ready to give you my contribution in a near future.
Congrats!
